tools/Tools.Common/Models/TypeMetadata.cs

```diff
@@ -217,6 +218,16 @@ public TypeMetadata(Type inputType)
 
         public List<MethodSignature> Constructors { get { return _constructors; } }
 
+        private string GetClassNameWithoutApiVersion(string className)
+        {
+            var matcher = Regex.Match(className, "([^\"]+)\\.Api[^\"]+\\.([^\"]+)");
```

Comment:
Adjust regex to make it specific to Azure PowerShell generated code. 
Microsoft.Azure.PowerShell.Cmdlets.*.Api